bento/src/main/java/com/yelp/android/bento/componentcontrollers/RecyclerViewComponentController.java

```diff
@@ -284,8 +286,29 @@ public void onScrolled(RecyclerView recyclerView, int dx, int dy) {
                         mComponentVisibilityListener.onScrolled();
                     }
                 };
+        mAdapterDataObserver =
```

Comment:
nit: Maybe change this method to `addVisibilityListeners` since we're doing more than one thing here.

Reply:
Makes sense!